.github/workflows/package-manager-ci.yml

```diff
@@ -28,6 +28,7 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node-version }}
+          check-latest: true
```

Comment:
Sometimes I like caches, for a non-critical task as this one or the `integration.yml`